src/SDKs/_metadata/monitor_data-plane.txt

```diff
@@ -0,0 +1,10 @@
+2017-09-22 22:25:47 UTC
+
```

Comment:
@vcanaa I don't see any data-plane in your SDK. Any reason this file is getting generated?

Reply:
Removing the data-plane folder and moving its specs to resource-manager is a work in progress. It is proposed in this PR (Swagger) https://github.com/Azure/azure-rest-api-specs/pull/1840, which I still see that PR as not merged.

The reason for moving the specs to resource-manager is that we really do not have any data-plane calls: everything is going through ARM. But due to historic reasons we had our specs in separate folders. When the last reorg of the repository was done, one of the folders was labeled as data-plane and the other as resource-manager. That configuration is now creating some issues for other teams that are helping us generate SDKs for other languages (Java for instance) because they have started the generation for the resource-manager folder only effectively leaving our metrics API (at least) out of their SDKs. That's why in PR 1840 I am proposing to move the specs to the resource-manager folder.